### PR TITLE
Fix trivial type mismatch

### PR DESCRIPTION
**Pull request by cheese**
Signed-off-by: Robin Lee <cheeselee@fedoraproject.org>

### CODE REVIEW

File: drivers/OCFSSR.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def detach(self, sr_uuid):
         except util.CommandException, inst:
                 raise xs_errors.XenError('OCFSUnMount', \
                       opterr='Failed to umount FS. Errno is %d' % \
-                      os.strerror(inst.code))
+                      inst.code)
```
Comment (MarkSymsCtx) [inline drivers/OCFSSR.py]: I'd lean to actually printing the stringified error message (or possibly both).

Also AFAIK, this code isn't actually used - it certainly isn't in the Citrix commercial product.

Comment (cheese) [reply drivers/OCFSSR.py]: Since inst.code is the exit status from subprocess, it is not generally mapped to string message by os.strerror.

**Comment by cheese** [comment]
Fixed the other two occurrences.

File: drivers/OCFSSR.py (hunk 1 above)
Comment (MarkSymsCtx) [inline drivers/OCFSSR.py]: If these aren't actually errnos then the message string is also incorrect. Replace Errno with Error is probably best.

**Pull request closed without merging**